source/guides/kernel/kernel-boot-params.rst

```diff
@@ -0,0 +1,81 @@
+.. _kernel_boot_params:
+
+Kernel boot parameters
+######################
+
```

Comment:
Suggest adding an opening sentence about what a kernel boot parameter is. Right now is immediately opens with a warning without context.

Also suggest linking to https://www.kernel.org/doc/Documentation/admin-guide/kernel-parameters.txt at some point

Reply:
Hold until after 09-06.